man/man8/zfs.8

```diff
@@ -487,6 +502,17 @@ been set somewhere in the tree under which the dataset resides.
 
 .sp
 .ne 2
+.mk
+.na
+\fB\fBkeystatus\fR
+.ad
+.sp .6
+.RS 4n
+Indicates if an encryption key is currently loaded into ZFS. The possible values are \fBnone\fR, \fBavailable\fR, and \fBunavaliable\fR. See \fBzfs key -l\fR and \fBzfs key -u\fR.
```

Comment:
It's really up to @behlendorf, but I find a lot of tools don't work great with super long lines.  The disadvantage to breaking them up is that when you make one change and reflow the text, every line shows up different.